Cargo.toml

```diff
@@ -63,7 +63,7 @@ eth_trie = "0.4.0"
 ethereum-types = "0.14.1"
 futures = "0.3.30"
 hashbrown = "0.14.5"
-hex = "0.4.3"
+hex = { version = "0.4.3", features = ["serde"] }
```

Comment:
I don't think this is needed?

Reply:
Use 
```
hex = { workspace = true }
```

Reply:
This is the main workspace config file @atanmarko